docs/api/commands/wait.mdx

```diff
@@ -16,6 +16,47 @@ cy.wait(alias, options)
 cy.wait(aliases, options)
 ```
 
+```typescript
+// Specifying request and response types for aliased intercepts
+type UserReq = {}
+type UserRes = {}
+type ActivityReq = {}
+type ActivityRes = {}
+
+cy.intercept('/users/*').as('getUsers')
+cy.intercept('/activities/*').as('getActivities')
+
+// As templated types:
+cy.wait<CustomRequest, CustomResponse>('@getUsers').then(
+  ({ request, response }) => {
+    request.body // will be of type UserReq
+    response.body // will be of type UserRes
+  }
+)
+
+// As inferred types, with type `Interception` available in `cypress/types/net-stubbing`
+cy.wait('@getUsers').then(
+  ({ request, response }: Interception<CustomRequest, CustomResponse>) => {
```

Comment:
Similarly to above, the types should match the above (and expected by the callback body):
`...Interception<UserReq, UserRes>...`